R/create.R

```diff
@@ -141,15 +141,9 @@ create_from_github <- function(repo,
   check_not_nested(destdir, repo)
   protocol <- match.arg(protocol)
 
-  repo <- strsplit(repo, "/")[[1]]
-  if (length(repo) != 2) {
-    stop(
-      code("repo"), " must be of form ",
-      value("owner/reponame"), call. = FALSE
-    )
-  }
-  owner <- repo[[1]]
-  repo <- repo[[2]]
+  ## TODO: do I need to do anything re: repo --> repo_spec?
```

Comment:
Do I need to do much/anything about this? `create_from_github()` is mostly for interactive use, probably has few users, `repo`/`repo_spec` is the first argument (so often unnamed) and partial matching works in our favour.

Reply:
I don't think you need to